src/SNLS_device_forall.h

```diff
@@ -8,37 +8,43 @@
 #ifndef SNLS_device_forall_h
 #define SNLS_device_forall_h
 
-#ifndef SNLS_GPU_THREADS
-#define SNLS_GPU_THREADS 256
+#ifndef SNLS_GPU_BLOCKS
+#define SNLS_GPU_BLOCKS 256
 #endif
 
 #include "SNLS_config.h"
 
 #if defined(SNLS_RAJA_PERF_SUITE)
+
+#include "SNLS_gpu_portability.h"
 #include "RAJA/RAJA.hpp"
 #include "chai/config.hpp"
 #include "chai/ExecutionSpaces.hpp"
 
+#include <variant>
```

Comment:
I don't see this include being used anywhere.  Am I missing something here?

Reply:
See lines 84-90:
```c++
 using rres = std::variant<
      rhost_res
#if defined(__snls_gpu_active__)
      ,
      rgpu_res
#endif
   >;
```

which is needed later on for when we need to supply the forall abstractions with the appropriate RAJA resource for the RAJA::forall to run on.